pimd/pim_nht.c

```diff
@@ -829,6 +832,9 @@ int pim_ecmp_nexthop_lookup(struct pim_instance *pim,
 		return 0;
 	}
 
+        memset(&nbrs, 0, sizeof(nbrs));
+        memset(&ifps, 0, sizeof(ifps));
```

Comment:
Tabs, not spaces please.

Reply:
Thank you, fixed (happened because of pasting buffers with the mouse instead of using vim's block copy, sorry)